opencti-platform/Dockerfile

```diff
@@ -1,47 +1,53 @@
-FROM node:12.21.0-alpine
+FROM node:12.21.0-alpine3.12 AS base
```

Comment:
Just tested succesfully with latest version of nodejs.
You can change that to 15.11.0-alpine3.13

Reply:
Thank you for the testing.
One question I do want to ask: What Node.js version shall we use for OpenCTI, is it `LTS (14.x)` or `Current (15.x)`?
And by updating the version we use in dockerfile. we should update all the configs in the repo like CircleCI & Drone CI config along with `package.json` 's engines property. Should I do it in this PR too?

Reference: https://nodejs.org/en/about/releases/

---
Update: I saw you just changed engines property in `package.json` to `>= 12.* <= 15.*`, so do you think we should stick to the Current Version (15.x)?

Reply:
Yeah i dont think we have any reason to not use the latest version.
We also of course need to change the CI to use the latest images

Reply:
Got it, I will use `15.11.0-alpine3.13` for the base image.